File: CS50x/cash/cash.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{
    int x;
    do
    {
        x = get_int("Change owed: ");
    }
    while (x < 0);
    int counter = 0;
    while (true)
    {
        if (x - 25 >= 0)
        {
            x = x - 25;
            counter++;
            continue;
        }
        else if (x - 10 >= 0)
        {
            x = x - 10;
            counter++;
            continue;
        }
        else if (x - 5 >= 0)
        {
            x = x - 5;
            counter++;
            continue;
        }
        else if (x - 1 >= 0)
        {
            x = x - 1;
            counter++;
            continue;
        }
        else if (x == 0)
        {
            break;
        }
    }
    printf("%i\n", counter);
}

File: CS50x/sentimental-cash/cash.py
from cs50 import get_float
while True:
    n = get_float("Change: ")
    if (n >= 0.0):
        break

n *= 100
count = int(n/25)
n %= 25
count += int(n/10)
n %= 10
count += int(n/5)
n %= 5
count += int(n/1)
print(count)
